scaleway/helpers.go

```diff
@@ -310,8 +315,8 @@ func regionComputedSchema() *schema.Schema {
 	}
 }
 
-// validateStringInSliceWithWarning helps to only returns warnings in case we got a non public locality passed
-func validateStringInSliceWithWarning(correctValues []string, field string) func(i interface{}, path cty.Path) diag.Diagnostics {
+// ValidateStringInSliceWithWarning helps to only returns warnings in case we got a non public locality passed
```

Comment:
Why is this change required?

Reply:
Not necessary anymore. I'll rollback this to private